app/database/backups/template.hbs

```diff
@@ -0,0 +1,37 @@
+<div class="row">
+  <div class="col-xs-8">
+    <div class="panel panel-default">
+      <div class="panel-heading">
+        <h3>Create a new database backup</h3>
+      </div>
+      <div class="panel-body">
+        <p>
+        You can initiate the creation of a new backup at any time. Existing
+        backups are shown below. Note that if you schedule a manual backup,
+        Aptible won't schedule an automated backup for another 24 hours.
+        </p>
+        <button class="btn btn-success btn-lg" {{action 'createBackup'}}>
+          Create new backup
```

Comment:
Let's use initial caps in button labels for consistency, e.g. "Create New Backup". I can put something in a style guide for this, if it would help.